pkg/specgenutil/volumes.go

```diff
@@ -446,7 +446,7 @@ func parseMountOptions(mountType string, args []string) (*universalMount, error)
 				return nil, fmt.Errorf("%v: %w", name, errOptionArg)
 			}
 			mnt.subPath = value
-		case "target", "dst", "destination":
+		case "target", "dst", "dest", "destination":
```

Comment:
Sorry but I have to nitpick here then. If you add this then it must be documented as well.